src/python/pants/backend/docker/goals/package_image_test.py

```diff
@@ -942,8 +935,7 @@ def test_get_context_root(
         tgt = DockerImageTarget({"context_root": context_root}, address)
         fs = DockerFieldSet.create(tgt)
         actual_context_root = fs.get_context_root(docker_options.default_context_root)
-        if expected_context_root:
-            assert actual_context_root == expected_context_root
```

Comment:
Should've caught this issue when I wrote this... alas I did not :P 

I did catch this issue now, however, when using the engine_error decorator on the visibility tests.